include/faabric/transport/PointToPointBroker.h

```diff
@@ -128,6 +131,18 @@ class PointToPointBroker
       groupFlags;
 
     std::shared_ptr<faabric::util::FlagWaiter> getGroupFlag(int groupId);
+
+    Message doRecvMessage(int groupId, int sendIdx, int recvIdx);
+
+    std::atomic<bool> orderMsg;
```

Comment:
Why does `orderMsg` need to be an `atomic<bool>` here? Are we modifying it from multiple threads?

In general it's best to name bools with getters and setters as `_isX` (variable), `isX()` (getter) and `setX()` (setter), in this case the following might be clearer:

- `_isMessageOrderingOn` (variable)
- `isMessageOrderingOn` (getter)
- `setMessageOrderingOn` (setter)